lektor/db.py

```diff
@@ -1,3 +1,11 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+from builtins import next
```

Comment:
builtins does not exist on Python 2 and these imports seem unnecessary or at least should be on the same line.  Why are those needed?